src/test/unit/mocks/opaPublicList.json

```diff
@@ -44,114 +140,210 @@
                                                     },
                                                     {
                                                         "caseUrn": "URN5678",
+                                                        "party": [
```

Comment:
This block of party information is identical to the previous block in the same case. The previous case already tested multiple defendants, you probably just need one defendant here rather than repeating the same test case.